docs/install.md

```diff
@@ -106,27 +126,38 @@ Once the module is installed and enabled, you can check its requirements on
 ![MongoDB on status page](images/mongodb-requirements.png)
 
 
-##Composer Requirements
+###Composer Requirements
 
-* Commands below are for those who are using composer already in your site to
-  manage module dependencies. To know more about composer [here][composer].
+* This section is applicable if you are using composer already in your site to
+  manage module dependencies. Know more about composer [here][composer].
 
 [composer]: https://www.drupal.org/docs/develop/using-composer/using-composer-to-manage-drupal-site-dependencies
 
-* At the root of your site
-  * If you are using the `drupal-composer/drupal-project` skeleton, just add
-    this package:
+  * At the root of your site
+    * If you are using the `drupal-composer/drupal-project` skeleton, just add
+      this package:
+
+         `composer require drupal/mongodb "^2.0.0"`
 
-        composer require drupal/mongodb "^2.0.0"
-  * Otherwise also add a composer requirement by typing:
+    * Otherwise also add a composer requirement by typing:
 
-        composer require drupal/mongodb "^2.0.0"
-        composer require mongodb/mongodb "^1.2.0"
-* If this is the first Composer dependency on the project, from the site root,
-  run:
+        `composer require drupal/mongodb "^2.0.0"`
+        `composer require mongodb/mongodb "^1.2.0"`
 
-        composer install
-* Enable the `mongodb` module. You now have access to the MongoDB services and
-  Drush commands.
+    * If this is the first Composer dependency on the project, run:
```

Comment:
Actually, just typing the previous "composer require" commands will already have installed the dependencies, so AFAICS this step is not needed.